js/models/ServerConfig.js

```diff
@@ -31,6 +34,31 @@ export default class extends BaseModel {
     };
   }
 
+  set(key, val, options = {}) {
+    // Handle both `"key", value` and `{key: value}` -style arguments.
+    let attrs;
+    let opts = options;
+
+    if (typeof key === 'object') {
+      attrs = key;
+      opts = val || {};
```

Comment:
lol... this is the second time you've flagged this code and this is actually code right from the Backbone source :)

The way set() works is that you could either call it like `model.set(attr, value, options)` or `model.set({ attr1: val, attr2: val}, options)`. This code facilitates that and ensures that even if the user is passing it in the first way, it will convert it to an object so the rest of the method just needs to handle the object format.

Reply:
I see. I'll make sure to read the entire Backbone source code and cross reference all previous reviews before each new code review. It may slow down the review process a little, but I'm sure it's worth it! ;)